<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreaterepoArguments</class>
 <widget class="QDialog" name="CreaterepoArguments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>211</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>461</width>
    <height>211</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>461</width>
    <height>211</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Createrepo Arguments</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/Icons/Images/yumrepomanager.png</normaloff>:/Icons/Images/yumrepomanager.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRestoreDefault">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore Default</string>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Icons/Images/Undo.png</normaloff>:/Icons/Images/Undo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>441</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>-1</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="labelWorkers">
        <property name="text">
         <string>Workers:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
        <property name="buddy">
         <cstring>spinBoxWorkers</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCacheDir">
        <property name="text">
         <string>CacheDir:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
        <property name="buddy">
         <cstring>lineEditCacheDir</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="spinBoxWorkers">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEditCacheDir"/>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBoxIncludeGroups">
        <property name="text">
         <string>Include Groups</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxCheckTimestamps">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Check Timestamps</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxIgnoreSymlinks">
        <property name="text">
         <string>Ignore Symlinks</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>144</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBoxNoDatabase">
        <property name="text">
         <string>No Database</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxUpdate">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxDeltas">
        <property name="text">
         <string>Deltas</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBoxXZCompression">
        <property name="text">
         <string>XZ Compression</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxPrettyFormat">
        <property name="text">
         <string>Pretty Format</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>checkBoxIncludeGroups</tabstop>
  <tabstop>checkBoxCheckTimestamps</tabstop>
  <tabstop>checkBoxIgnoreSymlinks</tabstop>
  <tabstop>checkBoxNoDatabase</tabstop>
  <tabstop>checkBoxUpdate</tabstop>
  <tabstop>checkBoxDeltas</tabstop>
  <tabstop>checkBoxXZCompression</tabstop>
  <tabstop>checkBoxPrettyFormat</tabstop>
  <tabstop>spinBoxWorkers</tabstop>
  <tabstop>lineEditCacheDir</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>pushButtonRestoreDefault</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
